### ISSUE TITLE
Should option out if newline(enter) will be counted as character count or not

### ISSUE THREAD

**Issue by vindecodex** [vindecodex/counting-str#7]
current functionality of `countCharacters()` was counting newline(enter) as characters no options.

**Comment by shubhdwiv12** [comment]
Hi vindecodex, can you explain this issue with little example, i will work on it

**Comment by vindecodex** [comment]
> Hi vindecodex, can you explain this issue with little example, i will work on it

`countCharacters()` should accept options arguments if newline or enter(key) should be counted as a character.

```
interface CountCharactersOption {
  withSpaces: boolean;
  withNewLine: boolean;
}
```
`countCharacters()` should check if `withNewLine` is `true` then enter key or newlines will be counted as characters.

**Comment by shubhdwiv12** [comment]
ok, i will be working on it, assign this issue to me.

**Comment by shubhdwiv12** [comment]
Hey @vindecodex , I have made the relevant changes but whenever i am running this make command to compile typescript file, it is showing the following error
$ make
tsc ./index.ts
process_begin: CreateProcess(NULL, tsc ./index.ts, ...) failed.
make (e=2): The system cannot find the file specified.
make: *** [Makefile:5: all] Error 2

any help which can resolve this issue, it is unable to find the index.ts file

**Comment by vindecodex** [comment]
:thinking: try to manually compile it without using the Makefile, command is -> `tsc ./index.ts`

run the command in the root directory of the project. :ok_hand: 

and have you install typescript in your machine ? just guessing :thinking:

**Comment by shubhdwiv12** [comment]
yeah, i have installed, it was some package issue, i removed and reinstalled, now its working fine, i have made some relevant changes in the code, its  showing merge conflict as i tweaked some old code, can you look at it , its my first open source contribution.

**Comment by vindecodex** [comment]
okay, there is two ways of fixing conflict, you can use vscode for it or through the main github page but I suggest vscode coz it has `accept both changes` clickable option. first pull from the main repository [this might help just try](https://github.com/vindecodex/gitguide#updating-forked-repo). after try to `git status` it will tell you what file got conflict open it through vscode, find the conflicts and just `accept both` i believe you didn't remove anything but only added something so just `accept both` I will just check it on your PR important thing no more conflict.
